client/src/configureStore.js

```diff
@@ -19,6 +20,7 @@ const sagaMiddleware = createSagaMiddleware();
 const configuredStore = createStore(persistedReducer, applyMiddleware(sagaMiddleware));
 sagaMiddleware.run(watchFetchMovies);
 sagaMiddleware.run(watchFetchMoviesOfTheSameGenre);
+sagaMiddleware.run(watchFetchMovie);
```

Comment:
_trivial_
It might be more convenient to introduce rootSaga that will be in charge of controlling other sagas e.g.

```javascript
// in sagas.js
import { all, fork } from 'reduxe/saga/effects';

export function* rootSaga() {
  yield all([
    fork(watchFetchMovies),
    fork(watchFetchMoviesOfTheSameGenre),
    fork(watchFetchMovie),
  ]);
}


// in configureStore.js
import createSagaMiddleware from 'redux-saga';
import { rootSaga } from './sagas';

const sagaMiddleware = createSagaMiddleware();
sagaMiddleware.run(rootSaga);
```